src/jupyter_contrib_nbextensions/nbextensions/code_font_size/READM.md

```diff
@@ -0,0 +1,4 @@
+Code Font Size
```

Comment:
this filename appears to have a typo - `READM.md` should probably be `README.md`?